### ISSUE TITLE
Are there hot keys for TimeControlNode?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/fourier-making-waves#92]
And if so, should they be in keyboard help for the Discrete screen?

**Comment by pixelzoom** [comment]
TimeControlNode has not had a comprehensive hot-key design. One of its subcomponent (play/pause button) supports a hot key by default, while the other subcomponents (step, etc.) do not support hot keys.  I think it would be odd to have partial support, so I'm going to turn off play/pause hot key until there's uniform support - see https://github.com/phetsims/scenery-phet/issues/682.

I've told @jessegreenberg that this is not blocking for Fourier 1.0 release.  @arouinfar do you agree?

**Comment by arouinfar** [comment]
I was confused by this statement because MAL does have a hotkey for StepForwardButton:
> One of its subcomponent (PlayPauseButton) supports a hotkey by default, while the other subcomponents (StepForwardButton, StepBackwardButton) do not support hotkeys.

But then I saw what @pixelzoom said in https://github.com/phetsims/scenery-phet/issues/682:
> And MAL added its own StepForwardButton support, which should be in TimeControlNode, see https://github.com/phetsims/scenery-phet/issues/681.

@pixelzoom I agree that this is not blocking, as the TimeControlNode is not critical to the learning goals of Fourier: Making Waves.

**Comment by pixelzoom** [comment]
Thanks @arouinfar. Marking this issue as deferred, to be addressed in the future (after 1.0).
